recipes/mimalloc/all/conanfile.py

```diff
@@ -0,0 +1,149 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MimallocConan(ConanFile):
+    name = "mimalloc"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/mimalloc"
+    description = "mimalloc is a compact general purpose allocator with excellent performance."
+    topics = ("conan", "mimalloc", "allocator", "performance", "microsoft")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "secure": [True, False],
+        "override": [True, False],
+        "single_object": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "secure": False,
+        "override": False,
+        "single_object": False
+    }
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt", "patches/*"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("mimalloc-" + self.version, self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
```

Comment:
Unfortunately, we have to do some stuff like checking the compiler version, if min_cppstd is not set.